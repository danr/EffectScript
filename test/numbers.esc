function main() {
  unit = ()
  "test:" + show(1+2*3) + ", " +
  "unit:" + show(unit)
}
