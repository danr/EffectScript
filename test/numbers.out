test:7, unit:()
